frontend/src/metabase/admin/upsells/UpsellUploads.tsx

```diff
@@ -0,0 +1,33 @@
+import { jt, t } from "ttag";
+
+import { getPlan } from "metabase/common/utils/plan";
+import { useSelector } from "metabase/lib/redux";
+import { getSetting } from "metabase/selectors/settings";
+
+import { UpsellCard } from "./components";
+
+export const UpsellUploads = ({ source }: { source: string }) => {
+  const plan = useSelector(state =>
+    getPlan(getSetting(state, "token-features")),
+  );
+
+  const showUpsell = plan === "oss" || plan === "starter";
+
+  if (!showUpsell) {
+    return null;
+  }
+
+  return (
+    <UpsellCard
+      title={t`Manage your uploads`}
+      campaign="manage-uploads"
+      buttonText={t`Try for free`}
+      buttonLink="https://www.metabase.com/upgrade"
+      source={source}
+    >
+      {jt`${(
```

Comment:
Could we add some context here to guide the translators?

Out of context, `_________ to manage your uploaded files...` might be puzzling. But I think if you provide the context string, `{0} is the string "Upgrade to Metabase Pro"`, that will help a good deal.

Reply:
I had a feeling this was coming.
Every time I see strings broken like this, I remember you. Saying this In the best possible way, for the record.

You opened my eyes to the plight of translators. Remembered it here as well but I guess I just blanked on what to provide as a context. Will do it. Thanks for the nudge and a hint.

Reply:
https://github.com/metabase/metabase/pull/52974/commits/f8da83edca827a0b100a81440489c4052d4a3751